libs/sdk-ui-charts/src/highcharts/chartTypes/warterfallChart/waterfallChartsSeries.ts

```diff
@@ -1,12 +1,27 @@
 // (C) 2023 GoodData Corporation
-import { IMeasureGroupDescriptor } from "@gooddata/sdk-model";
+import { IMeasure, IMeasureGroupDescriptor } from "@gooddata/sdk-model";
 import { DataViewFacade } from "@gooddata/sdk-ui";
 import { IColorStrategy, valueWithEmptyHandling } from "@gooddata/sdk-ui-vis-commons";
+import findLastIndex from "lodash/findLastIndex";
 
 import { isOneOfTypes, parseValue, unwrap } from "../_util/common";
 import { multiMeasuresAlternatingTypes } from "../_chartOptions/chartCapabilities";
 import { IUnwrappedAttributeHeadersWithItems } from "../../typings/mess";
 import { IPointData } from "../../typings/unsafe";
+import { ITotalConfig } from "../../../interfaces";
+
+function getTotalValue(pointDatas: IPointData[]) {
+    const lastTotalIndex = findLastIndex(pointDatas, (item: IPointData) => item.visible === false);
```

Comment:
As you may know, in a waterfall chart, the values always go up or go down if they have the same positive/negative values. But the total column always starts from 0. 
So, if a metric is total, we will add a shadow column that has value is the sum of all values before it, so that the total column can start from 0. This shadow column will be hidden on the chart. That's why it has `visible === false`. 
You can see this demo for more detail: https://jsfiddle.net/ndvan/9e5y2vtm/34/. The Positive Balance likes a metric set as total.

Reply:
yes, but visible== false flag can be set to whatever column manually, by clicking on corresponding legend item